darwin/datatypes.py

```diff
@@ -32,6 +32,37 @@ def get_sub(self, annotation_type: str) -> Optional[SubAnnotation]:
                 return sub
 
 
+@dataclass(frozen=True, eq=True)
+class VideoAnnotation:
+    annotation_class: AnnotationClass
+    frames: Annotation
+    keyframes: List[bool]
+    segments: List[List[int]]
+    interpolated: bool
+
+    def get_frame(self, frame_index):
+        return frames[frame_index]
+
+    def get_data(self, only_keyframes=True, post_processing=None):
+        if not post_processing:
+            post_processing = id
```

Comment:
I find this function a bit difficult to read (it might be because I'm not the most pythonic reviewer in the world).

However, I think it would be extremely beneficial to add a comment or two to explain why this `if` branch is needed.

Documenting the function parameters would also be a plus (e.g. what is `post_processing` supposed to do?)

Reply:
Maybe we can make the default value for post_processing id (identity function) in the function header, maybe that's clearer?